ResearchStack2UI/ResearchStack2UI/Styleguide/UIColor+BridgeKeyNames.swift

```diff
@@ -35,20 +35,30 @@ import UIKit
 
 extension UIColor {
     
-    /**
-     Override to set a primary tint color for the app.
-     */
-    @objc open class var primaryTintColor: UIColor? {
-        return nil
+    /// Override to set a primary tint color for the app.
+    @objc open class var primaryTintColor: UIColor {
+        return UIColor(hexString: "#5A478F")!
     }
     
-    /**
-     Override to set a secondary tint color for the app.
-     */
-    @objc open class var secondaryTintColor: UIColor? {
-        return nil
+    /// Override to set the "Light" primary tint color which can be used where the shade of the color should
+    /// be lighter than the primary color. This is used for selection state and the foreground on the
+    /// progress bar.
+    @objc open class var lightPrimaryTintColor: UIColor {
+        return UIColor(hexString: "#907FBA")!
+    }
+    
+    /// Override to set the "Dark" primary tint color which can be used where the shade of the color should
+    /// be darker than the primary color, but still in the same family. This is used for the background of
+    /// the progress bar and a selected checkbox.
+    @objc open class var darkPrimaryTintColor: UIColor {
+        return UIColor(hexString: "#332069")!
+    }
+    
+    /// Override to set a secondary tint color for the app.
+    @objc open class var secondaryTintColor: UIColor {
+        return UIColor(red: 1.0, green: 136.0 / 255.0, blue: 117.0 / 255.0, alpha: 1.0)
```

Comment:
seems a little odd to have rgba initializer here where we use hex colors above, but not a big deal